Tests/CacheWarmer/AspectCacheWarmerTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Go! AOP framework
+ *
+ * @copyright Copyright 2015, Lisachenko Alexander <lisachenko.it@gmail.com>
+ *
+ * This source file is subject to the license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Go\Symfony\GoAopBundle\Tests\CacheWarmer;
+
+use Go\Aop\Proxy;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use Symfony\Component\Filesystem\Filesystem;
+use Go\Symfony\GoAopBundle\Tests\TestProject\Application\Main;
+
+/**
+ * Class AspectCacheWarmerTest
+ */
+class AspectCacheWarmerTest extends WebTestCase
```

Comment:
Just curious shouldn't we move all these command tests to the framework itself or not? Because when they are in the bundle, they can be fragile due to changes in the framework.